DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -3661,11 +3785,22 @@ defaultproperties
     bingo_options(341)=(event="TiffanyHeli",desc="Rescue Tiffany Savage",max=1,missions=4096)
     bingo_options(342)=(event="AlarmUnitHacked",desc="Hack %s Alarm Sounder Panels",desc_singular="Hack 1 Alarm Sounder Panel",max=10)
     bingo_options(343)=(event="BuoyOhBuoy",desc="Buoy Oh Buoy (%s)",max=10,missions=94)
+    bingo_options(344)=(event="PlayerPeeped",desc="Despite everything, it's still you",max=1,missions=24446)
+    bingo_options(345)=(event="DangUnstabCond_peepedtex",desc="Condemned!",max=1,missions=2)
+    bingo_options(346)=(event="pa_TrainSign_D_peepedtex",desc="Closely inspect the Paris metro map",max=1,missions=2048)
+    bingo_options(347)=(event="IOnceKnelt",desc="I once knelt in this chapel",max=1,missions=2048)
+    bingo_options(348)=(event="GasCashRegister",desc="Check the cash register at the gas station",max=1,missions=4096)
+    bingo_options(349)=(event="LibertyPoints",desc="Visit the 11 points of the statue foundation",max=11,missions=2)
+    bingo_options(350)=(event="CherryPickerSeat",desc="Sit in the seat of a cherry picker",max=1,missions=49152)
+    bingo_options(351)=(event="ForkliftCertified",desc="Forklift Certified",max=1,missions=32770)
+    bingo_options(352)=(event="ASingleFlask",desc="Do you have a single flask to back that up? (%s)",max=10,missions=24190)
 
+    //Current bingo_options array size is 400.  Keep this at the bottom of the list as a reminder!
+//#endregion
```

Comment:
maybe?
```suggestion
//#endregion
    //#region Current bingo_options array size is 400.  Keep this at the bottom of the list as a reminder!
```

Reply:
I'm not sure that gives any value?  The comment is intended as a reminder as you add things on to the end of the list - keeping it as close to the options as possible is ideal, imo.